tests/SDKDataTest.php

```diff
@@ -10,12 +10,12 @@ final class SDKDataTest extends TestCase
     public function testGetVersion()
     {
         $data = new SDKData();
-        $this->assertEquals('1.3.0', $data->getSdkVersion());
+        $this->assertEquals('1.3.1', $data->getSdkVersion());
     }
 
     public function testGetSdkName()
     {
         $data = new SDKData();
-        $this->assertEquals('eppo-php-sdk', $data->getSdkName());
+        $this->assertEquals('eppo/php-sdk', $data->getSdkName());
```

Comment:
This will go into the URL parameters - these should be encoded but do you think the slash will cause issues with routing?

Reply:
we use http_build_query which urlencodes all of the parameter keys and values before appending to the URL.